syntax = "proto3";
package instance;




import "google/protobuf/struct.proto";

/*
CreateInstances请求
*/
message CreateInstancesRequest {
    /*
    模型Id
    */
    string objectId = 1 ;
    /*
    实例数据
    */
    repeated google.protobuf.Struct instances = 2 ;
}

/*
CreateInstances返回
*/
message CreateInstancesResponse {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    错误信息
    */
    string error = 2 ;
    /*
    返回消息
    */
    string message = 3 ;
    /*
    实例Id列表
    */
    repeated string data = 4 ;
}

/*
CreateInstancesApi返回
*/
message CreateInstancesResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    instance.CreateInstancesResponse data = 4 ;
}

